orbs/echoes/orb.yml

```diff
@@ -0,0 +1,134 @@
+version: 2.1
+
+description: |
+  Create Echoes 'release' using Echoes API, for Git repositories.
+
+executors:
+  default:
+    docker:
+      - image: "jobteaser/node"
```

Comment:
I'm wondering if we shouldn't use specific minimalist docker image for orbs, like Circle CI ones.

Don't see any specific element here (we could avoid use of `node` with `date` and `jq` command …).

We only need `curl` (`jq` installed with that) and `git`

Reply:
I guess we could. What benefits do you see of using another image ?

If we just end up replacing `node` by `jq`, I am not sure it's worth it. I find that sometimes a "good old" JS code is more easy to understand that a jq syntax (like what we would need on line 35 below).

Also I just checked and our **`jobteaser/node` image is almost 100Mo smaller that the base circleCI convenience image with just Ubuntu**.

Note that the only requirement today is to have a `bash` shell (an `sh` shell would not support some syntax used here - like bash arrays and $BASH_ENV). I tried using an alpine linux nodejs image but unfortunately is uses `sh` instead of `bash`.

Reply:
Oh sorry I mixed up `jobteaser/node` and `jobteaser/js-service` images 🤦.

So it's OK for me 🙏

Reply:
Yep sorry 🙇